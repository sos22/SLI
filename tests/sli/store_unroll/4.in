depth = 5
A true
B ordinary
C true
A -> B
B -> C
C -> A
B -> B
Roots:
A
End of roots
