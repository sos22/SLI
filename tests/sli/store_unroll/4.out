A true
B ordinary
C true
A2 dupe
B2 ordinary
C2 dupe
A3 dupe
B3 ordinary
C3 dupe
B4 ordinary
B5 ordinary
A -> B
B -> C
B -> B4
C -> A2
A2 -> B2
B2 -> C2
B2 -> B3
C2 -> A3
B3 -> C3
B4 -> C
B4 -> B5
B5 -> C
Roots:
A
End of roots
